src/renderer.test.ts

```diff
@@ -43,9 +43,18 @@ const imageElement = (): BodyElement =>
         role: none,
     });
 
-const imageElementWithRole = () =>
+const imageElementWithRole = (): BodyElement =>
     ({
-        ...imageElement(),
```

Comment:
Did you have an issue with the `BodyElement` return type conflicting with `imageElement`?